.chloggen/splunk_hec-http-server-close.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: splunkhecexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Close the HTTP servers used in testing
+
+# One or more tracking issues related to the change
+issues: [16285]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

Comment:
We don't need changelog entries for test changes. I'll merge this PR to unblock the tests, but please remove this file in another PR

Reply:
Submitted https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16318